src/utils.jl

```diff
@@ -133,6 +133,30 @@ function unpack_results(results_df)
 	return all_results
 end
 
+"""
+	bids_effects(model_df::DataFrame, effects_dict::Dict)
+
+Calculate [mariginalized effect](https://unfoldtoolbox.github.io/UnfoldDocs/Unfold.jl/stable/generated/HowTo/effects/) on all subjects found in the model dataframe using `effects_dict`.
+"""
+function bids_effects(model_df::DataFrame, effects_dict::Dict)
```

Comment:
this is counter to the effects function in Unfold, where the syntax is `effects(::Dict,::Model)`

I'm also thinking of whether we can argue to overload
```Unfold.effects(::Dict,::DataFrame)```

it is type-piracy, because we dont "own" `DataFrame` or `effects` though, but as long as we dont have an UnfoldResults-DataFrame, I'm not sure how to best handle it.

What do you think?